pkg/utl/rediscache/service_test.go

```diff
@@ -82,6 +86,32 @@ func TestGetUser(t *testing.T) {
 			},
 			want: testutls.MockUser(),
 		},
+		{
+			name: "CacheUserValueError",
```

Comment:
keep it uniform. Since we are using constant variable name, follow that approach only